src/app/services/user/user.service.ts

```diff
@@ -1,14 +1,20 @@
+import { map, first } from 'rxjs/operators';
 import { UserDto } from './../../data/modelDTO/user-dto';
 import { Injectable } from '@angular/core';
 import { AngularFirestore, AngularFirestoreDocument } from '@angular/fire/firestore';
+import { FirestoreService } from 'src/app/data/http/firestore.service';
+import { Observable } from 'rxjs';
+import { convertSnapshots } from 'src/app/data/utils/firestore-utils.service';
 
 @Injectable({
   providedIn: 'root'
 })
 export class UserService {
+  private USER_COLLECTION = 'user/';
 
   constructor(
-    private afs: AngularFirestore
+    private afs: AngularFirestore,
```

Comment:
please rename the variabe to be more readable - avoid using abbreviations for naming - the naming should be suggestive